src/index.js

```diff
@@ -123,26 +123,119 @@ export default class ToggleBlock {
 
   /**
    * Creates a toggle block view without paragraphs
+   * and sets the default content.
    */
   _createToggle() {
     this.wrapper = document.createElement('div');
     this.wrapper.classList.add('toggle-block__selector');
     this.wrapper.id = crypto.randomUUID();
 
     const icon = document.createElement('span');
+    const input = document.createElement('div');
+    const defaultContent = document.createElement('div');
 
     icon.classList.add('toggle-block__icon');
     icon.innerHTML = this.data.status === 'closed' ? toggleIconClosed : toggleIconOpen;
 
-    const input = document.createElement('div');
-
     input.classList.add('toggle-block__input');
     input.contentEditable = true;
-    input.addEventListener('keyup', this.createParagraphFromToggleRoot.bind(this));
     input.innerHTML = this.data.text || '';
 
+    // Events to create other blocks and destroy the toggle
+    input.addEventListener('keyup', this.createParagraphFromToggleRoot.bind(this));
+    input.addEventListener('keydown', this.removeToggle.bind(this));
+
+    // Establishes the placeholder for the toggle root when it's empty
+    input.addEventListener('keyup', this.setPlaceHolder.bind(this));
+    input.setAttribute('placeholder', 'Toggle');
+
+    // Calculates the number of toggle items
+    input.addEventListener('focus', this.calculateChildren.bind(this));
+    input.addEventListener('focusout', this.calculateChildren.bind(this));
+    input.addEventListener('focus', this.setDefaultContent.bind(this));
+    input.addEventListener('focusout', this.setDefaultContent.bind(this));
```

Comment:
@juliansg022 using a set of functions to establish related event listeners may be a good idea, what do you think?

Reply:
@juanhurtado10 That sounds good, I'll check it.